### PR TITLE
CAPV: move v1.6 release jobs to new preset

### PR DESCRIPTION
**Pull request by chrischdi**
This PR configures the new presets on the v1.6 release branch.

If this works we will revert the release-1.6 changes, remove the old presets and rename the new presets (remove the `-test` suffix)

### CODE REVIEW

File: config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presets.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  - name: VSPHERE_TLS_THUMBPRINT
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-vcenter-thumbprint
+  - name: VM_SSH_PUB_KEY
+    valueFrom:
+      secretKeyRef:
+        name: cluster-api-provider-vsphere-ci
+        key: vmc-e2e-vm-ssh-pubkey
```
Comment (sbueringer) [inline config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presets.yaml:29]:
```suggestion
        key: vmc-e2e-vm-ssh.pubkey
```
Maybe?

**Comment by sbueringer** [comment]
I think otherwise it should be all correct

File: config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presets.yaml (hunk 1 above)
Comment (chrischdi) [reply config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presets.yaml]: nice catch 👍

**Comment by sbueringer** [comment]
/lgtm
/approve

Let's give it a try :)

**Comment by sbueringer** [comment]
Ah good point, we push the image in any case we just don't use it

**Pull request merged**